content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -81,7 +81,7 @@ Value       | Description
 `Pending`   | The Pod has been accepted by the Kubernetes cluster, but one or more of the containers has not been set up and made ready to run. This includes time a Pod spends waiting to be scheduled as well as the time spent downloading container images over the network.
 `Running`   | The Pod has been bound to a node, and all of the containers have been created. At least one container is still running, or is in the process of starting or restarting.
 `Succeeded` | All containers in the Pod have terminated in success, and will not be restarted.
-`Failed`    | All containers in the Pod have terminated, and at least one container has terminated in failure. That is, the container either exited with non-zero status or was terminated by the system.
+`Failed`    | All containers in the Pod have terminated, and at least one container has terminated in failure. That is, the container either exited with non-zero status or was terminated by the system, and will not be restarted.
```

Comment:
```suggestion
`Failed`    | All containers in the Pod have terminated, and at least one container has terminated in failure. That is, the container either exited with non-zero status or was terminated by the system, and is not set for automatic restarting.
```

Reply:
Hi Tim, I used that sentence because it's consistent with Line 83, and changing it may be confusing for a new user. If consistency is not a priority and a minor variety is okay, kindly let me know, and I will commit the suggestion.